uniflow-android/src/main/java/io/uniflow/android/livedata/LiveDataPublisher.kt

```diff
@@ -30,15 +30,15 @@ open class LiveDataPublisher(
             UniFlowLogger.debug("$tag new state $state")
             _currentState = state
             if (pushStateUpdate) {
-                UniFlowLogger.debug("$tag --> $state")
+                UniFlowLogger.logState(state)
```

Comment:
Should I move this above `if (pushStateUpdate)` 🤔

Reply:
above, you already have `UniFlowLogger.debug("$tag new state $state")`. It was just to help debug that we have pushed the state

Reply:
The only reason why I'm pushing this is for better visibility when we are setting state or event, this way we can create our specific logger and push more information this will help us to investigate what could go wrong, also it will avoid pushing too much unsessesery information because we are using debug in many different places.